specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/machineLearningServices.json

```diff
@@ -608,15 +597,64 @@
             }
           }
         }
+      },
+      "post": {
+        "tags": [
+          "OperationalizationClusters",
+          "MachineLearningComputes"
+        ],
+        "operationId": "MachineLearningCompute_SystemUpdate",
+        "description": "System Update On Machine Learning compute.",
+        "x-ms-examples": {
+          "System Update Compute": { "$ref": "./examples/systemUpdateCompute.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Compute System Update initiated.",
```

Comment:
Is this and update for the `MachineLearningCompute` resource?
`Update` is usually a `PATCH` operation. Did this service undergo an API review?

Reply:
Also, 202 corresponds to `Accepted` which ideally should be a long running operation